app/stac_api/admin.py

```diff
@@ -213,6 +238,12 @@ class Media:
     list_display = ['name', 'collection', 'collection_published']
     list_filter = [CollectionFilterForItems]
 
+    #helper function which displays the bytes in human-readable format
+    def displayed_total_data_size(self, instance):
+        return filesizeformat(instance.total_data_size)
+
+    displayed_total_data_size.short_description = 'Total data size'
```

Comment:
pls add a unit here (`B`, `KB`?)

Reply:
I dont quite understand why that is necessary, doesn't the unit from the filesizeformat suffice?
![Screenshot from 2024-10-22 11-41-21](https://github.com/user-attachments/assets/7eab98cb-91bd-4e3e-90c2-b06dbd9688dc)